Automattic/newspack-plugin
# Plugin Footer

### All Submissions:






### Changes proposed in this Pull Request:



This PR removes the old school Settings page and add a footer to the Wizards to replace it.

* Newspack version now takes you to the Updates wizard
* About: newspack.pub
* Documentation: newspack.pub/support/
* Component Demo
* Setup Wizard
* Reset Newspack
* Reset WPcom authentification

Closes https://github.com/Automattic/newspack-plugin/issues/811

Add the `WP_NEWSPACK_DEBUG` flag to `wp-config.php` to see most of the footer items.

### How to test the changes in this Pull Request:

1. Switch to this branch
2. Check if you see the new footer
3. Test the different links

### Other information:
